nominations/tests.py

```diff
@@ -1,3 +1,87 @@
 from django.test import TestCase
+from models import Application
+from local_groups.models import Group
 
-# Create your tests here.
+
+class ApplicationTestCase(TestCase):
+    def setUp(self):
+        # create group
+        group = Group.objects.create(
```

Comment:
Is this really editing db or is this mocked somehow? For unit test stuff like this should be mocked.

Reply:
It's mocked. https://docs.djangoproject.com/en/1.8/topics/testing/overview/#the-test-database

Reply:
Sounds like it is a real db that is separate from the regular app db?
```
Tests that require a database (namely, model tests) will not use your “real” (production) database. Separate, blank databases are created for the tests.

Regardless of whether the tests pass or fail, the test databases are destroyed when all the tests have been executed.

You can prevent the test databases from being destroyed by using the test --keepdb option. This will preserve the test database between runs. If the database does not exist, it will first be created. Any migrations will also be applied in order to keep it up to date.

The default test database names are created by prepending test_ to the value of each NAME in DATABASES. When using SQLite, the tests will use an in-memory database by default (i.e., the database will be created in memory, bypassing the filesystem entirely!). The TEST dictionary in DATABASES offers a number of settings to configure your test database. For example, if you want to use a different database name, specify NAME in the TEST dictionary for any given database in DATABASES.

On PostgreSQL, USER will also need read access to the built-in postgres database.

Aside from using a separate database, the test runner will otherwise use all of the same database settings you have in your settings file: ENGINE, USER, HOST, etc. The test database is created by the user specified by USER, so you’ll need to make sure that the given user account has sufficient privileges to create a new database on the system.
```